packages/mongo-nav/src/project-nav/ProjectNav.spec.tsx

```diff
@@ -77,7 +83,7 @@ describe('packages/mongo-nav/src/project-nav', () => {
           constructProjectURL={constructProjectURL}
           urls={urlFixtures}
           activeProduct="cloud"
-          hosts={hostDefaults}
+          hosts={hostDefaults(false)}
```

Comment:
This should follow the `environment` flag so we can test the created values/

Reply:
is this what you mean?
```
 environment="commercial"
          hosts={hostDefaults()}```

Reply:
`hostDefaults(environment === 'government')`

Reply:
here we want to test by default but i updated MongoNav to have tests for these new created values

Reply:
https://github.com/mongodb/leafygreen-ui/pull/501/files#diff-832b8f9837f5879d08b9fddbcc69af38R399